### DISCUSSION TITLE
how to claim github pack for 2021 passouts

### DISCUSSION THREAD

**Discussion by ghost** [github/india#555] [Students]
my BE is completed in this year . so i dont have id card to claim this and during virtual internship it is clearly mentioned that it is for 2021, 2022 students . but problem is how 2021 passouts going to claim student pack in github?

**Comment by HemantSachdeva** [comment]
As this is only for Students and you are no longer having a valid identity proof that can prove you are a student then they are not going to approve your request. You can still try [applying](https://education.github.com/pack) for Student Developer Pack by sending them your expired ID proof pic.

**Comment by TheSuranaverse** [comment]
A group for all to get notified for any `Official GitHub Externship Announcement`, `Discuss & Clear any Doubts`, `Learn & Complete` together.
One-stop place for daily `Events/Workshops` + `Hackathons` + `Internships/Jobs` + `Free Scholarship, Courses, Swags & Goodies` + `24/7 Live Chat and Mentors to help you with your Doubts`  + `24/7 Live Music` & much more...

Link to join: https://discord.gg/ceMXzhfaka

Thank you 🙏
